test/tla/Bug1682.tla

```diff
@@ -1,20 +1,44 @@
 ---- MODULE Bug1682 ----
 
-EXTENDS Integers
+EXTENDS Integers, Variants
+
+(*
+ @typeAlias: MESSAGE =
+      RequestVote({ term: Int, from: Str })
+    | Vote({ from: Str, to: Str, term: Int })
+    | AppendEntries({ term: Int, from: Str }) ;
```

Comment:
Not sure I like the `({ ... })` double delimiter syntax in practice.

Reply:
You can have any type there, `{ ... }` is a record